Aspire Budgeting/PreviewProvider/MockProvider.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MockProvider.swift
+//  Aspire Budgeting
+//
+
+import Foundation
+
+struct MockProvider {
+  static var tabBarItems: [TabBarItem] {
+    var tabBarItems = [TabBarItem]()
+    tabBarItems.append(TabBarItem(imageName: "rectangle.grid.1x2", title: "Dashboard"))
+    tabBarItems.append(TabBarItem(imageName: "creditcard", title: "Accounts"))
+    tabBarItems.append(TabBarItem(imageName: "arrow.up.arrow.down", title: "Transactions"))
+    tabBarItems.append(TabBarItem(imageName: "gear", title: "Settings"))
+    return tabBarItems
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/PreviewProvider/MockProvider.swift#L17"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>MockProvider.swift:17</code></td>
    </tr>
  </tbody>
</table>